skcosmo/utils/orthogonalizers.py

```diff
@@ -25,26 +25,36 @@ def X_orthogonalizer(x1, c=None, x2=None, tol=1e-12):
     :param c: index of the column to orthogonalize by
     :type c: int, less than m
 
-    :param x2: a separate set of columns to orthogonalize by
+    :param x2: a separate set of columns to orthogonalize with column-by-column
```

Comment:
I feel like this is missing the end of the sentence? 

Overall I don't quite understand what this function is doing just by reading the doc. Do you think you could add a bit of explanations?